modules/shops/client.lua

```diff
@@ -106,6 +106,46 @@ client.shops = setmetatable(data('shops'), {
 								distance = target.distance or 2.0
 							})
 						end
+					elseif shop.peds then
+						for i = 1, #shop.peds do
+							local ped = shop.peds[i]
+							local shopid = type..'-'..i
+							id += 1
+
+							shops[id] = {
+								zoneId = shopid,
+								blip = blip and createBlip(blip, ped.loc)
+							}
+							local pedmodel = joaat(ped.ped)
+							RequestModel(pedmodel)
+							while not HasModelLoaded(pedmodel) do
+								Wait(0)
+							end
+							local PedId = CreatePed(0, ped.ped, ped.loc.x, ped.loc.y, ped.loc.z-1, ped.loc.w, true, true)
```

Comment:
Why is the peds here Networked? that means if multiple people join shops will be messed up

Reply:
cause i'm dumb sorry x)